platform/android/build.gradle

```diff
@@ -1,7 +1,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.4.0'
+    ext.kotlin_version = '1.6.10'
```

Comment:
Is there some feature that requires this bumping up version?

Reply:
Yeah the serializer, but we decided to go back to 1.4 by using a different one (which is already in use)